README.md

```diff
@@ -1,3 +1,7 @@
+[![Travis CI](https://travis-ci.org/Microsoft/sqltoolsservice.svg?branch=dev)](https://travis-ci.org/anthonydresser/sqltoolsservice)
```

Comment:
We'll change this to https://travis-ci.org/**Microsoft**/sqltoolsservice once this is checked in / created, correct?